### PR TITLE
Modin-plugin example

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
Added examples link
Minor changes to README file
Signed-off-by: SmritiSatyanV <smriti@union.ai>

### CODE REVIEW

File: plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def _flyte_schema(self, dataset) -> str:
                 dataset.remote_path, self._local_file_path, is_multipart=True
             )
 
-        # DataFrame (Pandas, Spark, etc.)
+        # DataFrame (pandas, Spark, etc.)
```
Comment (AlekhyaSasi) [inline plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py:143]: how about 'S'park?

File: plugins/flytekit-pandera/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Flytekit Pandera Plugin
 
-Flytekit python natively supports many data types, including a FlyteSchema type for type-annotating Pandas DataFrames. The Flytekit Pandera plugin provides an alternative for defining DataFrame schemas by integrating with Pandera, which is a runtime data validation tool for Pandas DataFrames.
+Flytekit python natively supports many data types, including a FlyteSchema type for type-annotating pandas DataFrames. The Flytekit Pandera plugin provides an alternative for defining DataFrame schemas by integrating with Pandera, which is a runtime data validation tool for pandas DataFrames.
```
Comment (AlekhyaSasi) [inline plugins/flytekit-pandera/README.md:3]:
```suggestion
Flytekit python natively supports many data types, including a FlyteSchema type for type-annotating pandas DataFrames. The Flytekit Pandera plugin provides an alternative for defining DataFrame schemas by integrating with Pandera, a runtime data validation tool for pandas DataFrames.
```

File: requirements.in [hunk 3]
```diff
@@ -7,7 +7,7 @@ attrs<21
 # https://github.com/flyteorg/flyte/issues/1732.
 jsonschema<4
 pyyaml<6
-# A number of dependencies, including pandas and numpy, started dropping support for python 3.7 in their latest
-# releases (pandas>=1.4.0 and numpy>=1.22.0). More details in https://github.com/flyteorg/flyte/issues/2115.
+# A number of dependencies, including pandas and NumPy, started dropping support for Python 3.7 in their latest
```
Comment (AlekhyaSasi) [inline requirements.in:10]: Is the version # right?

File: plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py (hunk 1 above)
Comment (SmritiSatyanV) [reply plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py:143]: The [documentation](https://spark.apache.org/docs/latest/) uses uppercase 'S'.

File: requirements.in (hunk 3 above)
Comment (SmritiSatyanV) [reply requirements.in:10]: This isn't something I am sure about, since I only changed the word from 'numpy' to 'NumPy'.

**Review by AlekhyaSasi — DISMISSED** [review]

File: requirements.in [hunk 4]
```diff
@@ -7,7 +7,7 @@ attrs<21
...
+# releases (pandas>=1.4.0 and NumPy>=1.22.0). More details in https://github.com/flyteorg/flyte/issues/2115.
 pandas<1.4.0
-numpy<1.22.0
+NumPy<1.22.0
```
Comment (samhita-alla) [inline requirements.in:13]: Can we revert the changes in this file except for `Python`? `pandas and NumPy` somewhat looks odd, and `numpy` is the library to be installed; we need not modify its case.

File: plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py (hunk 1 above)
Comment (samhita-alla) [reply plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py:143]: This looks odd, @SmritiSatyanV. Let's revert it or have `pandas, spark`.

File: doc-requirements.txt [hunk 5]
```diff
@@ -118,14 +114,18 @@ numpy==1.22.2
...
     # via flytekit
 poyo==0.5.0
     # via cookiecutter
 protobuf==3.19.4
     # via
     #   flyteidl
     #   flytekit
+    #   googleapis-common-protos
+    #   protoc-gen-swagger
+protoc-gen-swagger==0.1.0
```
Comment (samhita-alla) [inline doc-requirements.txt:127]: Can you make sure your changes are in sync with the master branch because this library is already available in master?

File: requirements.in (hunk 3 above)
Comment (samhita-alla) [reply requirements.in:10]: Let it be.

File: requirements.in (hunk 4 above)
Comment (SmritiSatyanV) [reply requirements.in:13]: Sure

File: plugins/flytekit-modin/README.md [hunk 6]
```diff
@@ -1,11 +1,12 @@
 # Flytekit Modin Plugin
 
-Modin is an emerging drop-in replacement or rather extension of Pandas. This plugin could be helpful to use Modin as a data type.
+Modin is a Python extension for pandas that helps handle large datasets. It is a light-weight extension that is similar to pandas API. It uses the concept of parallelism to reduce overhead, and improve performance of pandas operations by leveraging the compute resources available. This plugin could help use Modin as a data type.
```
Comment (samhita-alla) [inline plugins/flytekit-modin/README.md:3]:
```suggestion
Modin is a pandas-accelerator that helps handle large datasets. It is a light-weight extension that is similar to the pandas API. It uses the concept of parallelism to reduce overhead, and improve the performance of pandas operations by leveraging the compute resources available. 

This plugin supports Modin as a data type.
```

File: plugins/flytekit-modin/setup.py [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=0.22.0,<1.0.0", "modin", "ray", "fsspec"]
+plugin_requires = ["flytekit>=0.22.0,<1.0.0", "modin>=0.13.0", "modin", "ray", "fsspec"]
```
Comment (samhita-alla) [inline plugins/flytekit-modin/setup.py:7]: "modin" is repeated twice.

File: requirements.in [hunk 8]
```diff
@@ -2,12 +2,12 @@
 -e file:.#egg=flytekit
 attrs<21
 # We need to restrict constrain the versions of both jsonschema and pyyaml because of docker-compose (which is
-# used to run integration tests) pins those 2 libraries. We are in the process of removing docker-compose in
-# favor of a more generic solution that involves flytectl to stand-up the sandbox, described in
+# used to run integration tests) pins those two libraries. We are in the process of removing docker-compose in
+# favor of a more generic solution that involves Flytectl to stand-up the Sandbox, described in
```
Comment (samhita-alla) [inline requirements.in:6]:
```suggestion
# favor of a more generic solution that involves Flytectl to stand-up the sandbox, described in
```
We needn't capitalize "S" in sandbox.

File: plugins/flytekit-modin/README.md [hunk 9]
```diff
@@ -1,11 +1,13 @@
 # Flytekit Modin Plugin
 
-Modin is an emerging drop-in replacement or rather extension of Pandas. This plugin could be helpful to use Modin as a data type.
+Modin is a pandas-accelerator that helps handle large datasets. It is a light-weight extension that is similar to the pandas API. It uses the concept of parallelism to reduce overhead, and improve the performance of pandas operations by leveraging the compute resources available. This plugin could help use Modin as a data type.
+
+This plugin supports Modin as a data type.
```
Comment (samhita-alla) [inline plugins/flytekit-modin/README.md:5]: This and the last line of the previous paragraph mean the same.

File: plugins/flytekit-modin/requirements.txt [hunk 10]
```diff
@@ -2,212 +2,308 @@
 # This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile requirements.txt
```
Comment (samhita-alla) [inline plugins/flytekit-modin/requirements.txt:5]: You'll have to pip-compile the requirements.in file only. Can you please run the command again?

File: requirements.txt [hunk 11]
```diff
@@ -2,186 +2,275 @@
 # This file is autogenerated by pip-compile with python 3.8
```
Comment (samhita-alla) [inline requirements.txt:2]: Let's not modify this file.

File: plugins/flytekit-modin/requirements.txt (hunk 10 above)
Comment (SmritiSatyanV) [reply plugins/flytekit-modin/requirements.txt:5]: Why would it be requirements.txt? The master branch also says pip-compile requirements.in https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-modin/requirements.txt @samhita-alla

**Review by samhita-alla — APPROVED** [review]
